test/vulnz-check/vulnz_check.yaml

```diff
@@ -0,0 +1,293 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: vulnerability-checking-
+  annotations: {pipelines.kubeflow.org/kfp_sdk_version: 1.3.0, pipelines.kubeflow.org/pipeline_compilation_time: '2021-02-02T16:00:14.924668',
+    pipelines.kubeflow.org/pipeline_spec: '{"description": "A pipeline to check vulnerability
+      for all Kubeflow Pipelines images", "inputs": [{"default": "1.3.0", "name":
+      "version", "optional": true, "type": "String"}, {"default": "gcr.io/gongyuan-pipeline-test/dev",
+      "name": "registry_url", "optional": true, "type": "String"}], "name": "Vulnerability
+      Checking"}'}
+  labels: {pipelines.kubeflow.org/kfp_sdk_version: 1.3.0}
+spec:
+  entrypoint: vulnerability-checking
+  templates:
+  - name: fetch-image-digest
+    container:
+      args: [--image, '{{inputs.parameters.kfp-images-Output-loop-item}}', '----output-paths',
+        /tmp/outputs/Output/data]
+      command:
+      - sh
+      - -ec
+      - |
+        program_path=$(mktemp)
+        echo -n "$0" > "$program_path"
```

Comment:
Can you please use the latest SDK version for the compilation? The latest one uses `printf "%s" "$0"` instead of `echo -n "$0"`